### ISSUE TITLE
There is now way to track future activities

### ISSUE THREAD

**Issue by RobertChrist** [monicahq/monica#195]
Let's say I'm going hiking with a contact one month from now.

If I add this event as an activity in the future, it doesn't show up in my "What's coming up" feed, because the app assumes that activities have already happened.

If I add it as a reminder, there is no field for the date of the actual activity.  The best I can do is say, give me a reminder 29 days from now, that "I'm going hiking tomorrow".

And tasks don't have dates associated with them.  So there is no way for me to track this.

**Request:** Ideally, I think the way this should work, is I should be able to add an activity with a future date.  Because it is a future date, this activity should show up in my "Whats coming up" feed.  Additioanlly, If I add a future date, the app should ask me if I would like to set a reminder, which would get added to that contact's reminders.

If we can build all of the previous, then it would be really handy for the dashboard to have a link to a calendar, so I can see what events I have coming up in the future.

**Comment by ewwhite** [comment]
I’m checking in to see if there has been any movement on this, or perhaps a better way to show future activities/events?

**Comment by RobertChrist** [comment]
@ewwhite I opened this case in 2017, and haven't heard almost anything since.  Its been referenced in a few other issues.  I, frankly, am extremely surprised.  I don't see any way in which this is a useable product until this feature is implemented / issue is fixed.

**Comment by ewwhite** [comment]
This is a bit upsetting since I've populated data already and just realized the functionality was missing. It would be good to hear the development roadmap/stance on this request.

**Comment by binarytide** [comment]
this is a missing killer feature, I almost setup an instance for my wife but without this feature there just ins't enough utility for our needs

**Comment by alexander** [comment]
Agreed. Although I can use a calendar to follow up on activities. It would be great to have this on the dashboard as well.

**Comment by RobertChrist** [comment]
It is now 2023, and I circled back to check on this issue.  It is still missing from the application, unfortunately :(
